modules/opm-cli-ref-serve.adoc

```diff
@@ -36,7 +36,7 @@ $ opm serve <source_path> [<flags>]
 |Enables debug logging.
 
 |`h`, `--help`
-|Help for serve.
+|Help output for the `serve` command.
```

Comment:
This is the default output for cobra CLI subcommands.  I'm sure we could change this instance, but I think that's a slippery slope we should only traverse if required, and then with synchronous code/doc PRs.

Reply:
Same re: https://github.com/openshift/openshift-docs/pull/59181#discussion_r1188727603.